### DISCUSSION TITLE
Error  starting wildfly

### DISCUSSION THREAD

**Discussion by mery8** [Keyfactor/ejbca-ce#739] [Q&A]
Hi,

First of all, I want to thank this community for the great help.
I’m trying to install EJBCA. I deployed the ejbca.ear file on WildFly and then restarted the server. However, I encountered the following error: 
![image](https://github.com/user-attachments/assets/ad6b820d-7edb-4308-acc1-966707ffa711)
<img width="960" alt="error" src="https://github.com/user-attachments/assets/ee919804-cc1b-43ae-8b57-8b24ee5dffd2" />

<img width="546" alt="modilexml" src="https://github.com/user-attachments/assets/e93478cd-ffe3-4c3c-af92-f27896c87d68" />

Could anyone help me understand what might be causing this issue and how to resolve it? Thank you in advance!

**Comment by primetomas** [comment]
You don't seem to have added a working database driver correctly:
https://docs.keyfactor.com/ejbca/8.3.2/wildfly-26#id-(8.3.2)WildFly26-AddDatabaseDriver

Or it doesn't match the datasource you configured:
https://docs.keyfactor.com/ejbca/8.3.2/wildfly-26#id-(8.3.2)WildFly26-AddaDatasource

**Comment by mery8** [comment]
Thank you. You pointed me in the right direction.
